README.md

```diff
@@ -66,10 +66,13 @@ ran.
 However if there are issues downloading the images (e.g. proxy) they can be built 
 within this repo:
 ```
-version=$(python -c "import armory; print(armory.__version__)")
-docker build --build-arg armory_version=${version} --target armory-tf1 -t twosixarmory/tf1:${version} .
-docker build --build-arg armory_version=${version} --target armory-tf2 -t twosixarmory/tf2:${version} .
-docker build --build-arg armory_version=${version} --target armory-pytorch -t twosixarmory/pytorch:${version} .
+bash docker/build.sh
+```
+If only a specific target image is desired, run the relevant lines in `docker/build.sh`.
+NOTE: if the repo is pulled from master instead of pip installed, and is currently on a non-release branch,
+you will instead need to run:
+```
```

Comment:
Perhaps safer to throw an error in `build.sh` if the version contains `dev`? Then we could instruct to use build-dev since they're working from master

Reply:
`build.sh` now throws and error and exits if the version contains `-dev`.